tasks/system_probe.py

```diff
@@ -1501,3 +1501,16 @@ def test_microvms(
     ctx.run(
         f"cd ./test/new-e2e && go run ./scenarios/system-probe/main.go --name usama-saqib-test {go_args} --shutdown-period 720",
     )
+
+
+@task
+def generate_git_file_diff(ctx):
+    file_diff = ctx.run(
+        "git diff --name-only $(git merge-base $(inv release.get-release-json-value base_branch) HEAD) HEAD", hide=True
+    )
+    file_diff_lines = file_diff.stdout.split("\n")
+    for k, line in enumerate(file_diff_lines):
+        if k != 0:
+            print("\\n", end="")
+        print(line, end="")
```

Comment:
Is this just re-adding the newlines that were removed with the `split("\n")`?

Reply:
No exactly it's replacing ``\n`` by `\\n`` but I was doing something else at the same time didn't I could just've used replace direclty. Fixed :+1